tools/shared/README.md

```diff
@@ -11,122 +11,243 @@ Shared mixins, tools, etc. that support developing Spectrum Web Components.
+                <sp-table-cell>Anchor-like properties and rendering</sp-table-cell>
+            </sp-table-row>
+            <sp-table-row>
+                <sp-table-cell>FocusVisiblePolyfillMixin</sp-table-cell>
+                <sp-table-cell>Mixin</sp-table-cell>
+                <sp-table-cell>Polyfill for :focus-visible support</sp-table-cell>
+            </sp-table-row>
+            <sp-table-row>
+                <sp-table-cell>ObserveSlotPresence</sp-table-cell>
+                <sp-table-cell>Mixin</sp-table-cell>
+                <sp-table-cell>Observe presence of slotted content</sp-table-cell>
+            </sp-table-row>
+            <sp-table-row>
+                <sp-table-cell>ObserveSlotText</sp-table-cell>
+                <sp-table-cell>Mixin</sp-table-cell>
+                <sp-table-cell>Observe text changes in slots</sp-table-cell>
+            </sp-table-row>
+        </sp-table-body>
+    </sp-table>
+</div>
+
+### Utilities
+
+#### getDeepElementFromPoint
 
 The `getDeepElementFromPoint` method allows you to obtain the deepest possible element at a given coordinates on the current page. The method will step into any available `shadowRoot`s until it reaches the first element with no `shadowRoot` or no children available at the given coordinates.
 
-### Focusable
+**When to use:** Use this when you need to find the actual target element at specific coordinates, especially when working with shadow DOM where `document.elementFromPoint()` might not give you the deepest element.
 
-The `Focusable` subclass of `LitElement` adds some helpers method and lifecycle coverage in order to support passing focus to a container element inside of a custom element. The Focusable base class handles tabindex setting into shadowed elements automatically and is based heavily on the [aybolit delegate-focus-mixin](https://github.com/web-padawan/aybolit/blob/master/packages/core/src/mixins/delegate-focus-mixin.js).
+```javascript
+import { getDeepElementFromPoint } from '@spectrum-web-components/shared';
+
+const element = getDeepElementFromPoint(x, y);
+```
+
+#### getActiveElement
+
+Use this helper to find an `activeElement` in your component.
+
+**When to use:** Use this when you need to determine which element currently has focus, especially in components with shadow DOM where `document.activeElement` might not give you the correct element.
+
+```javascript
+import { getActiveElement } from '@spectrum-web-components/shared';
+
+const activeEl = getActiveElement(this);
+```
+
+### Base classes
+
+#### Focusable
+
+The `Focusable` subclass of `SpectrumElement` adds helper methods and lifecycle coverage to support passing focus to a container element inside of a custom element. The Focusable base class handles tabindex setting into shadowed elements automatically and is based heavily on the [aybolit delegate-focus-mixin](https://github.com/web-padawan/aybolit/blob/master/packages/core/src/mixins/delegate-focus-mixin.js).
```

Comment:
Nit: We could add backticks to the `Focusable` in the second sentence, just to be consistent.